backend/app/services/task_dispatcher.py

```
@@ -0,0 +1,444 @@
+        if auth_token:
+            headers["Authorization"] = f"Bearer {auth_token}"
+
+        # Propagate request_id for distributed tracing
+        try:
+            from shared.telemetry.context import get_request_id
+
+            request_id = get_request_id()
+            if request_id:
+                headers["X-Request-ID"] = request_id
+        except Exception:
+            pass  # Ignore if telemetry module not available
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)